client/components/eligibility-warnings/domain-warning/index.tsx

```diff
@@ -0,0 +1,23 @@
+import { __ } from '@wordpress/i18n';
+import { ReactElement } from 'react';
+import Card from '../card';
+import InfoLabel from '../info-label';
+
+type DomainEligibilityWarningProps = {
+	wpcomDomain: string | null;
+	stagingDomain: string | null;
+};
+
+const DomainEligibilityWarning = ( {
+	wpcomDomain,
+	stagingDomain,
+}: DomainEligibilityWarningProps ): ReactElement => (
+	<Card title={ __( 'New Store Domain' ) }>
+		<InfoLabel label={ __( 'New' ) }>{ stagingDomain }</InfoLabel>
+		<p>
+			{ `By installing this product your subdomain will change. Your old subdomain (${ wpcomDomain }) will no longer work. You can change it to a custom domain on us at anytime in future.` }
```

Comment:
Do we need some translation in here ?

Reply:
👍 yes, I copied the code from https://github.com/Automattic/wp-calypso/pull/58527. Let me udpate it.

Reply:
addressed [here](https://github.com/Automattic/wp-calypso/pull/58719/commits/258d37664da01f5a2bd886b7a1a4afee7a07562d). Thanks, Rolando. it was a nice one.